dao/src/main/java/se/fortnox/reactivewizard/db/query/parts/CollectionOptionsQueryPart.java

```diff
@@ -36,11 +41,33 @@ public void visit(StringBuilder sql, Object[] args) {
             if (collectionOptions != null) {
                 if (collectionOptions.getSortBy() != null) {
                     String sortBy = CamelSnakeConverter.camelToSnake(collectionOptions.getSortBy());
-                    for (String allowed : queryAnnotation.allowedSortColumns()) {
-                        if (allowed.equals(sortBy)) {
-                            collectionOptionsHasOrderBy = true;
-                            addOrderBy(sql, buildOrderBy(collectionOptions.getOrder(), allowed));
-                            break;
+                    var sortByParts = sortBy.split(",");
+                    if (sortByParts.length == 1) {
+                        for (String allowed : queryAnnotation.allowedSortColumns()) {
+                            if (allowed.equals(sortBy)) {
+                                collectionOptionsHasOrderBy = true;
+                                addOrderBy(sql, buildOrderBy(collectionOptions.getOrder(), allowed));
+                                break;
+                            }
+                        }
+                    } else {
+                        Collections.reverse(Arrays.asList(sortByParts));
+                        for (var sortByPart: sortByParts) {
```

Comment:
Would be nice to filter out the columns that have already been appended to the ordering. For instance, sortBy="name asc, id desc, name asc, id desc, ..." should not become "ORDER BY name ASC, id DESC, name ASC, id DESC, ..." but rather "ORDER BY name ASC, id DESC".

Reply:
Would make the code more complicated and would not change the result. It is also a fault made by the user of RW and should not occur.

Reply:
I agree that it would make the code more complicated and would not change the output. I am not sure I understand what you mean that it is a fault made by the user of RW and should not occur. Do you mean that the responsibility of dealing with "sortby=id asc, name desc, id asc, name desc, ..." falls on the application developer?

Even though it would make the code more complicated and doesn't change the output (as well as probably a slight performance impact), I prefer to see the filtering because I find it unreasonable that we would allow an end user to craft a query that looks like that. Furthermore, while it may not be a security concern related to postgres, I have other security concerns that we can discuss in private.